drivers/nrf_wifi/fw_if/umac_if/inc/fmac_ap.h

```diff
@@ -13,8 +13,9 @@
 #define __FMAC_AP_H__
 
 #include "host_rpu_data_if.h"
+#ifndef NRF70_OFFLOADED_RAW_TX
```

Comment:
Why is this needed. Ideally this `fmac_ap.h` should not get included in the offloaded raw TX build. Same comment for some of the other files too (e.g. `fmac_peer.h` etc)

Reply:
It's included from event file, I will remove this and add change in event file.